client/common/services/canvas.service.js

```diff
@@ -21,7 +21,6 @@
             context.fillStyle = vertexColor;
             context.beginPath();
             context.arc(xPos, yPos, size, 0, 2 * Math.PI);
-            context.closePath();
```

Comment:
This is unnecessary for circles since it basically draws another circle on top of the original.